eox_hooks/__init__.py

```diff
@@ -5,7 +5,7 @@
 
 import django.dispatch
 
-__version__ = '6.2.0'
+__version__ = '6.2.1'
```

Comment:
Supporting a new Open edX Release should be a minor. ref. https://github.com/eduNEXT/eox-theming/pull/55#discussion_r1688085628

```suggestion
__version__ = '6.3.0'
```

Reply:
Thanks @BryanttV